src/main/java/service/GitLabService.java

```diff
@@ -400,15 +404,18 @@ private MergeRequestResult acceptAutoMergeRequest(String gitlabEventUUID, MergeR
 	private MergeRequest assignMrToCascadeResponsible(String gitlabEventUUID, MergeRequest mr) {
 		final Long mrNumber = mr.getIid();
 		final Long project = mr.getProjectId();
-		Long cascadeResponsibleId = getCascadeResponsible(gitlabEventUUID, project, getPrevMergeRequestNumber(mr.getSourceBranch()));
-		if (cascadeResponsibleId != null) {
-			Log.infof("GitlabEvent: '%s' | Assigning MR '!%d' to cascade responsible with id '%d'", gitlabEventUUID, mrNumber, cascadeResponsibleId);
+		List<Long> cascadeResponsibleIds = getCascadeResponsibleIds(gitlabEventUUID, project, getPrevMergeRequestNumber(mr.getSourceBranch()));
+		if (cascadeResponsibleIds != null) {
+			Log.infof("GitlabEvent: '%s' | Assigning MR '!%d' to cascade responsible(s) with id(s): '%s'",
+					gitlabEventUUID,
+					mrNumber,
+					cascadeResponsibleIds.stream().map(String::valueOf).collect(Collectors.joining("', '")));
```

Comment:
Isn't the logger displaying correctly a `List<Long>` already? In that case you do not need this conversion with the steam api.

Reply:
Good point, yes! Thank you, I changed it accordingly.